app/models/article.rb

```diff
@@ -1,5 +1,7 @@
 class Article < ApplicationRecord
   belongs_to :user
+  include Commentable
+  has_many :comments, as: :commentable
```

Comment:
dependent destroy. https://guides.rubyonrails.org/association_basics.html

Reply:
why is this here if the code is also in the concern? just move this to the concern

Reply:
need this, doesnt work otherwise

Reply:
can move this to the concern and also add dependent destroy